mava/components/building/best_checkpointer.py

```diff
@@ -6,12 +7,17 @@
 
 from mava.components.component import Component
 from mava.core_jax import SystemBuilder, SystemParameterServer
+from mava.utils.lp_utils import termination_fn
 
 
 @dataclass
 class BestCheckpointerConfig:
     checkpointing_metric: Tuple = ("mean_episode_return",)
     checkpoint_best_perf: bool = False
+    # Flag to calculate the absolute metric
+    absolute_metric: bool = False
+    # How many episodes to run evaluation for
+    absolute_metric_duration: Optional[int] = 320
 
```

Comment:
Is there any way we could have this be something like `evaluation_duration * 10` by default?

Reply:
We can set it as default in case someone didn't define the absolute metric duration :raised_hands:

Reply:
Thank you :raised_hands: